src/components/connect-create-buttons.tsx

```diff
@@ -0,0 +1,65 @@
+import { __ } from '@wordpress/i18n';
+import { cx } from '../lib/cx';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { ArrowIcon } from './arrow-icon';
+import Button, { ButtonVariant } from './button';
+import offlineIcon from './offline-icon';
+import Tooltip from './tooltip';
+
+interface ConnectCreateButtonsProps {
+	connectSite: () => void;
+	isOffline: boolean;
+	connectButtonVariant: ButtonVariant;
+	createButtonVariant: ButtonVariant;
+	disableConnectButtonStyle?: boolean;
+}
+
+export const ConnectCreateButtons = ( {
+	connectSite,
+	isOffline,
+	createButtonVariant,
+	connectButtonVariant,
+	disableConnectButtonStyle,
+}: ConnectCreateButtonsProps ) => {
+	return (
+		<>
+			<Tooltip
+				disabled={ ! isOffline }
+				text={ __( 'Connecting a site requires an internet connection.' ) }
+				icon={ offlineIcon }
+			>
+				<Button
+					onClick={ connectSite }
+					disabled={ isOffline }
+					aria-disabled={ isOffline }
+					variant={ connectButtonVariant }
+					className={ cx(
+						! disableConnectButtonStyle &&
+							! isOffline &&
+							'!text-a8c-blueberry !shadow-a8c-blueberry'
+					) }
+				>
+					{ __( 'Connect site' ) }
+				</Button>
+			</Tooltip>
+			<Tooltip
+				disabled={ ! isOffline }
+				text={ __( 'Creating a site requires an internet connection.' ) }
+				icon={ offlineIcon }
+			>
+				<Button
+					onClick={ () => {
+						getIpcApi().openURL( 'https://wordpress.com/start/new-site' );
+					} }
+					variant={ createButtonVariant }
+					className={ cx( ! isOffline && '!text-a8c-blueberry !shadow-a8c-blueberry' ) }
+					disabled={ isOffline }
+					aria-disabled={ isOffline }
+				>
+					{ __( 'Create new site' ) }
+					<ArrowIcon />
+				</Button>
+			</Tooltip>
+		</>
+	);
```

Comment:
I decided to extract these two buttons in the common component as we are using them in two places and the code is quite repetitive